pkg/istio-agent/xds_proxy.go

```diff
@@ -686,23 +686,6 @@ func (p *XdsProxy) getTLSOptions(agent *Agent) (*istiogrpc.TLSOptions, error) {
 	}, nil
 }
 
-// sendUpstream sends discovery request.
-func sendUpstream(upstream xds.DiscoveryClient, request *discovery.DiscoveryRequest) error {
-	return istiogrpc.Send(upstream.Context(), func() error { return upstream.Send(request) })
-}
-
-// sendDownstream sends discovery response.
-func sendDownstream(downstream adsStream, response *discovery.DiscoveryResponse) error {
-	tStart := time.Now()
-	defer func() {
-		// This is a hint to help debug slow responses.
-		if time.Since(tStart) > 10*time.Second {
-			proxyLog.Warnf("sendDownstream took %v", time.Since(tStart))
```

Comment:
@ramaraochavali This kind of info is helpful and no harm

Reply:
ok. I can add that part back if agree on removing the timeout